app/vite.config.js

```diff
@@ -140,7 +140,7 @@ function directusExtensions() {
 	const prefix = '@directus-extensions-';
 	const virtualIds = APP_EXTENSION_TYPES.map((type) => `${prefix}${type}`);
 
-	let extensionEntrys = {};
+	let extensionEntries = {};
```

Comment:
`extensionEntrys` is an abbreviation for `extensionEntrypoints`. Not sure which linguistic rules apply in such a case, but the former feels more correct to me. It doesn't really matter too much, though 🤷‍♂️

Reply:
There's no way I would have figured that out, I'm happy to make a PR to expand it to `extensionEntrypoints`.

Reply:
I'd appreciate that! Sounds like a good idea to be more explicit in this case 👍